nodes/parachain/src/command.rs

```diff
@@ -40,6 +38,31 @@ use sp_core::hexdisplay::HexDisplay;
 use sp_runtime::traits::Block as BlockT;
 use std::{io::Write, net::SocketAddr};
 
+trait IdentifyChain {
+	fn is_peregrine(&self) -> bool;
+	fn is_spiritnet(&self) -> bool;
+}
+
+impl IdentifyChain for dyn sc_service::ChainSpec {
+	fn is_peregrine(&self) -> bool {
+		self.id().starts_with("kilt_peregrine")
+	}
+	fn is_spiritnet(&self) -> bool {
+		self.id().starts_with("kilt_spiritnet")
```

Comment:
does this also work for existing chain specs?

Reply:
It does not. Should be fixed in [a5e9d01](https://github.com/KILTprotocol/mashnet-node/pull/319/commits/a5e9d01128c1c5eb2c0761278f41fe1ac282911d)